packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -113,3 +116,163 @@ class DefaultTaggedText extends StatelessWidget {
     );
   }
 }
+
```

Comment:
maybe move this widget over to its own file under src/flutter. It is complex enough to be worth its own file. I would propose keeping comm_widgets.dart for widgets that don't really have complex business logic but are good to share.

Reply:
SGTM, moved.